docs/src/pages/css-in-js/basics/basics.md

```diff
@@ -126,6 +126,113 @@ export default withStyles(styles)(HigherOrderComponent);
 
 {{"demo": "pages/css-in-js/basics/HigherOrderComponent.js"}}
 
+## Nesting selectors
```

Comment:
I have a concern with the verbosity of documenting the 3 API alternatives every time. What do you think of sticking to the hook one? We will have to document more subtilities like this one, I make sense now to explain that the 3 APIs support the same features. Internally withStyles is build with makeStyles, styled is build with withStyles (we should probably refactor that, it would be better to use makeStyles for styled)

Reply:
Makes sense, it definitely seems repetitive. I'll remove the other alternatives and just use hook

Reply:
awesome 👍